pkg/packagekit/autoupdates.jsx

```diff
@@ -397,21 +407,32 @@ export class AutoUpdates extends React.Component {
         );
 
         return (<>
-            <Button variant="secondary"
-                    isDisabled={!enabled}
-                    onClick={() => {
-                        if (!this.state.backend.installed) {
-                            install_dialog(this.state.backend.packageName)
-                                    .then(() => {
-                                        this.initializeBackend(true);
-                                        this.setState({ showModal: true });
-                                    }, () => null);
-                        } else {
-                            this.setState({ showModal: true });
-                        }
-                    }}>
-                {_("Edit")}
-            </Button>
+            <Split hasGutter>
+                <SplitItem>
```

Comment:
So here you use the split item to nicely position a label - value pair. 
I suggest you use DescritionList for such usecases. If for some reason you can't, (I see there are some buttons on the right) let's make sure that the generated html is also accessible for screen readers. For that you will need an aria-labelledby  on the second split item.